addon/mixins/copyable.js

```diff
@@ -154,14 +154,14 @@ export default Mixin.create({
       ) {
         let value = this.get(name);
 
-        if (Copyable && Copyable.detect(value)) {
+        if ((Copyable && Copyable.detect(value)) || (value && value._isFragment)) {
```

Comment:
Are all model fragments copyable?

Reply:
They are. For some reason Copyable.detect didn't seem to work.